doc/source/e2e-ptl-guide.rst

```diff
@@ -0,0 +1,172 @@
+Using Ray with Pytorch Lightning
```

Comment:
title needs to the row length or else compiler will complain

Reply:
Fixed!